### PR TITLE
wire up the ui to the new search api

### PR DESCRIPTION
**Pull request by scottlepp**
Ref https://github.com/grafana/search-and-storage-team/issues/153

### CODE REVIEW

File: pkg/storage/unified/resource/resource.proto [hunk 1]
```diff
@@ -399,6 +399,9 @@ message ResourceSearchRequest {
 
   // explain each result (added to the each row)
   bool explain = 9;
+
+  // filters
+  repeated string filters = 10;
```
Comment (ryantxu) [inline pkg/storage/unified/resource/resource.proto:404]: the ListOptions already has typed filters for labels and fields...  This lets us avoid needing to parse the filter

**Review by ryantxu — APPROVED** [review]

File: pkg/apis/dashboard/v0alpha1/search.go [hunk 2]
```diff
@@ -69,9 +69,9 @@ type DashboardHit struct {
 	// The display nam
 	Title string `json:"title"`
 	// Filter tags
-	Tags []string `json:"tags,omitempty"`
+	Tags []string `json:"tags"`
 	// The UID/name for the folder
-	Folder string `json:"folder,omitempty"`
+	Folder string `json:"location"`
```
Comment (ryantxu) [inline pkg/apis/dashboard/v0alpha1/search.go:74]: Any reason to change from folder?

**Review by ryantxu — APPROVED** [review]

File: pkg/storage/unified/resource/resource.proto (hunk 1 above)
Comment (scottlepp) [reply pkg/storage/unified/resource/resource.proto]: 👍  updated

File: pkg/apis/dashboard/v0alpha1/search.go (hunk 2 above)
Comment (scottlepp) [reply pkg/apis/dashboard/v0alpha1/search.go:74]: the front end wants it to be `location` which is outside of the searcher implementation.

Comment (scottlepp) [reply pkg/apis/dashboard/v0alpha1/search.go:74]: i had to map `name` to `title` so i could do it on the front end if you want to keep this as `folder`

**Pull request merged**